### PR TITLE
i18n: Minor translation corrections

### PR DESCRIPTION
**Pull request by exterkamp**
**Summary**
Some minor corrections to :fr: :india: :sri_lanka:  translations.

### CODE REVIEW

File: lighthouse-core/lib/i18n/locales/fr.json [hunk 1]
```diff
@@ -447,7 +447,7 @@
...
+    "message": "Les ingénieurs en navigation recommandent que les pages contiennent moins de 1 500 éléments DOM environ. La zone d'écoute idéale est une profondeur d'arborescence inférieure à 32 éléments et contenant moins de 60 éléments enfant/parent. Un grand DOM peut accroître l'utilisation de la mémoire, entraîner de plus longs [calculs de style](https://developers.google.com/web/fundamentals/performance/rendering/reduce-the-scope-and-complexity-of-style-calculations) et des [redispositions] coûteuses (https://developers.google.com/speed/articles/reflow). [En savoir plus](https://developers.google.com/web/tools/lighthouse/audits/dom-size)."
```
Comment (exterkamp) [inline lighthouse-core/lib/i18n/locales/fr.json]: `[redispositions] coûteuses (https://developers.google.com/speed/articles/reflow).`

"redispositions" is disconnected from its link...and it doesnt seem to play nice with the report...
image

Comment (exterkamp) [reply lighthouse-core/lib/i18n/locales/fr.json]: Made #9000 to track this.

Comment (brendankenny) [reply lighthouse-core/lib/i18n/locales/fr.json]: can you fix manually for now? We can't ship this

Comment (brendankenny) [reply lighthouse-core/lib/i18n/locales/fr.json]: (I'm glad you noticed because I don't think I would have :)

Comment (exterkamp) [reply lighthouse-core/lib/i18n/locales/fr.json]: Fixed!

image

**Pull request merged**